contributing-tips.rst

```diff
@@ -0,0 +1,142 @@
+
+If you have a question, ask. Don't guess.
+
+.. note::
+
+  Many new contributors don't ask questions because they are worried about appearing under informed. Please set this worry aside.
+
+  **You will never be judged harshly for asking clarifying questions or for seeking more information.** 
+
+.. _if-you-get-stuck:
+
+If you get stuck while working
+--------------------------------
+
+- Ask for help in the issue comments. Maybe you can get back on track and complete the issue.
+
+  - Asking questions is always better than guessing.
+
+- Submit a ``[WIP]`` (work in progress) pull request.
+
+  - If we can see what progress you have made, it is easier to offer help.
+  - Even if you don't complete the project, perhaps someone else can pull in your in-progress work and build on it.
+  - Sometimes your in-progress work is an improvement over not having it, and so we'll merge in something even if it isn't complete.
+
+- If you really cannot move forward, it is okay to abandon an issue.
+
+.. _abandon-issue:
+
+It is okay to abandon an issue
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes you simply cannot complete work you have said you were going to complete. This could happen because you don't have the skills or knowledge to complete the work, or because the issue cannot actually be completed as scoped, or because you don't have the time.
+
+Please let the community know in the issue discussion. This way, everyone knows that someone else can take up the project (or that we need to rethink it).
+
+If you did significant work on a project before abandoning it, consider filing a ``[WIP]`` (work in progress) PR, so that others can see what you did and potentially build off of it.
+
+.. _issue-takes-long-time:
+
+If an issue takes a long time to complete
+-------------------------------------------
+
+For our purposes, a "long time" is a week or more, from when you first announce your intention to work on something until submitting a merge-ready PR.
+
+An issue might take a long time because:
+
+- it is complex and requires lots of hours 
+- you only have a short period of time each day to work on it
+- you are new to the project and are having to learn as you go
+
+The thing that matters is: **Are you actively working on the issue, and making progress, at least a little bit?**
+
+If you are actively working on it, we do not want someone else to jump on and try to work on it at the same time. So please keep the community informed of your work by filing a ``[WIP]`` (work in progress) PR and comitting to it as you work.
+
+.. _issues-only:
+
+Issues only
+----------------
+
+All PRs must be directly connected to open issues. PRs should not represent suggestions, good ideas, or independent intiative. 
+
+If you have a good idea, file an issue. If you are curious about whether something should be an issue, chat with one of the core team in the `#docs-code` channel on the `Slack <https://opendatakit.slack.com>`_.
```

Comment:
Adding *wait for a collaborator or admin to comment on the new issue before you start working on it* can also help. Since people open issues themselves and submit a PR before the issue is validated

Reply:
This is a very good point.

Reply:
I think mentioning the forum is also important. Involved in the forum kind of keeps you in touch with what's actually going on in other projects (new release of Collect, updates on Outreachy etc)